lms/djangoapps/bulk_enroll/views.py

```diff
@@ -60,7 +60,6 @@ class BulkEnrollView(APIView):
     def post(self, request):
         serializer = BulkEnrollmentSerializer(data=request.data)
         if serializer.is_valid():
-            request.POST = request.data
```

Comment:
It appears they are:
```
ipdb> id(request)
140610886734992
ipdb> id(self.request)
140610886734992
```

but it also appears that `request.POST` is the same as `request.data`:
```
ipdb> p request.POST
<QueryDict: {u'action': [u'enroll'], u'courses': [u'org.1/course_1/Run_1'], u'identifiers': [u'robot+test+5@edx.org'], u'email_students': [u'False']}>
ipdb> p request.data
<QueryDict: {u'action': [u'enroll'], u'courses': [u'org.1/course_1/Run_1'], u'identifiers': [u'robot+test+5@edx.org'], u'email_students': [u'False']}>
ipdb> id(request.POST)
140610886285712
ipdb> id(request.data)
140610886285712
```
So I'm unsure if there's a reason why this line exists.